docs/release-notes/26_2_0.adoc

```diff
@@ -0,0 +1,9 @@
+Today marks a significant milestone in the evolution of Keycloak JS with the release of version `26.2.0`. This new version represents a shift in how the JavaScript adapter develops and evolves alongside the Keycloak ecosystem. Although this new version does not bring any functional changes to the adapter compared to previous versions, it does come with several organizational changes users should be made aware of.
+
+The most notable change is that Keycloak JS now breaks free from the main Keycloak project's release cycle. As https://www.keycloak.org/2025/01/keycloak-2610-released#_dedicated_release_cycle_for_the_node_js_adapter_and_javascript_adapter[announced earlier this year], the JavaScript adapter will follow its own independent development path. The separation from the main project allows for more frequent releases of features, bugfixes, and improved responsiveness to community feedback. The JavaScript adapter will continue to be backwards compatible with all actively supported releases of the Keycloak server, and deviation from this will be considered a breaking change.
```

Comment:
```suggestion
The most notable change is that Keycloak JS now breaks free from the main Keycloak project's release cycle. As https://www.keycloak.org/2025/01/keycloak-2610-released#_dedicated_release_cycle_for_the_node_js_adapter_and_javascript_adapter[announced earlier this year], the JavaScript adapter will follow its own independent development path. The separation from the main project allows for more frequent releases of features, bug fixes, and improved responsiveness to community feedback. The JavaScript adapter will continue to be backwards compatible with all actively supported releases of the Keycloak server, and deviation from this will be considered a breaking change.
```